src/app/layout.tsx

```diff
@@ -0,0 +1,27 @@
+import type { Metadata } from 'next'
+import { Inter } from 'next/font/google'
+import React from 'react'
+import { Header } from '../shares/header'
+import './globals.css'
+
+const inter = Inter({ subsets: ['latin'] })
+
+export const metadata: Metadata = {
+  title: 'Online store',
+  description: 'Online store',
+}
+
+export default function RootLayout({
+  children,
+}: {
+  children: React.ReactNode
+}) {
+  return (
+    <html lang="en">
```

Comment:
is it a good practice to put html and body here? As I know it's a good practice to put those tags into separate document file. RootLayout component should only include the part of the layout that's common to all pages so far